presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -896,7 +899,11 @@ public ScalarFunctionImplementation getScalarFunctionImplementation(Signature si
             return specializedScalarCache.getUnchecked(getSpecializedFunctionKey(signature));
         }
         catch (UncheckedExecutionException e) {
-            throw Throwables.propagate(e.getCause());
+            if (e.getCause() instanceof ByteCodeException) {
+                throw new PrestoException(StandardErrorCode.NOT_SUPPORTED, e.getCause());
```

Comment:
- Because the error message is in MethodDefinition, it is specific to method (not function) and thus the limit can only be 254 (I made a mistake by typing 255). I'm not sure if it is hurtful for let user know what their problem is when hitting this. (Removed it anyway)
- BytecodeException is specifically created for error thrown in bytecode project and needs to be caught and re-throw as PrestoException. I'm not quite understanding why other case could go through this if statement?